src/base/searchengine.cpp

```diff
@@ -669,13 +670,20 @@ PluginVersion SearchEngine::getPluginVersion(QString filePath)
     if (!plugin.open(QIODevice::ReadOnly | QIODevice::Text))
         return {};
 
+    const PluginVersion invalidVersion;
+
     PluginVersion version;
     while (!plugin.atEnd()) {
         QByteArray line = plugin.readLine();
         if (line.startsWith("#VERSION: ")) {
             line = line.split(' ').last().trimmed();
-            version = PluginVersion::tryParse(line, {});
-            qDebug() << "plugin" << filePath << "version: " << version;
+            version = PluginVersion::tryParse(line, invalidVersion);
+            if (version == invalidVersion) {
+                LogMsg(tr("Search plugin '%1' contains invalid version string ('%2')")
+                    .arg(QFileInfo(filePath).fileName()).arg(QString::fromUtf8(line)), Log::MsgType::WARNING);
```

Comment:
>Do you want me to replace QFileInfo with Utils::Fs::fileName()?

I agree that we don't have enough info about cost. And the only reason I want you to replace it is just so we have uniform usage across the source.
If you're busy, I can merge as-is.